id_monkey_patches/id-ui-latlon-display.js

```diff
@@ -0,0 +1,237 @@
+// A huge copy/paste swath that adds a lat/lon display element to the bottom
```

Comment:
Where's it being copied from?

Reply:
Oh, right -- that's not very obvious is it?

The whole `id_monkey_patches/` directory is full of overrides of the iD source code (found in `vendor/`). In this case, it's source code from the original `iD.ui` object.

Reply:
ah got it thanks :)